conf/messages.cy

```diff
@@ -89,7 +92,7 @@ do-you-want-to-opt-out.h1=A ydych am analluogi grwpiau mynediad?
 do-you-want-to-opt-out.no.label=Na, rwyf am gadw grwpiau mynediad wedi’u galluogi
 do-you-want-to-opt-out.yes.error=Dewiswch ‘Iawn’ i analluogi grwpiau mynediad
 do-you-want-to-opt-out.yes.label=Iawn, rwyf am analluogi grwpiau mynediad
-ending.in=ending in {0}
+ending.in=*****{0}
```

Comment:
Will ask the design team about this 👀 since I don't think it's good for a11y.

replacing "ending in" with "*****" is going to be worse for screenreaders